@kiva/kv-shop/src/components/KvPaymentSelect.vue

```diff
@@ -0,0 +1,351 @@
+	border: 0;
+}
+
+/* Credit Card payment content */
+/* Main payment method content - either button or form */
+/* Classes for input fields. - Credit Card Only */
+/* Payment Method Form wrapper */
+.kv-payment-select [data-braintree-id="card"] .braintree-sheet__content.braintree-sheet__content--form {
+	padding: 0;
+}
+/* Form error colors */
+.kv-payment-select [data-braintree-id="card"] .braintree-sheet__content .braintree-form__field-error {
+	@apply tw-text-danger tw-text-left tw-font-medium;
+}
+/* Remove extra left padding from form field/label wrapper */
+.kv-payment-select [data-braintree-id="card"] .braintree-sheet__content .braintree-form__field-group {
+	padding-left: 0;
+	margin-bottom: 1.25rem;
+}
+/* Invalid form field */
+.kv-payment-select [data-braintree-id="card"] .braintree-sheet__content
+.braintree-form__field-group.braintree-form__field-group--has-error .braintree-form__hosted-field {
+	@apply tw-bg-danger/[0.15] tw-border-danger tw-rounded-sm;
+}
+/* Form fields */
+.kv-payment-select [data-braintree-id="card"] .braintree-sheet__content
+.braintree-form__field-group:not(.braintree-form__field-group--has-error)
+.braintree-form__field:not(.braintree-form__checkbox) .braintree-form__hosted-field {
+	@apply tw-text-tertiary tw-bg-secondary tw-rounded-sm tw-border-secondary;
+}
+/* Help text next to form label. */
+.kv-payment-select [data-braintree-id="card"] .braintree-sheet__content span.braintree-form__descriptor {
+	display: none;
+}
+/* Exp, CVV, and Postal Code groups */
+.kv-payment-select [data-braintree-id="card"]
+.braintree-sheet__content [data-braintree-id="expiration-date-field-group"],
+.kv-payment-select [data-braintree-id="card"]
+.braintree-sheet__content [data-braintree-id="cvv-field-group"],
+.kv-payment-select [data-braintree-id="card"]
+.braintree-sheet__content [data-braintree-id="postal-code-field-group"] {
+	width: 49%;
+	flex-basis: auto;
+	flex-grow: unset;
+}
+
+/* Credit Card icons in form header */
+.kv-payment-select [data-braintree-id="card-view-icons"] {
+	padding-bottom: 0;
+}
+.kv-payment-select [data-braintree-id="card-view-icons"] > div {
+	padding: 0;
+	border: 1px solid #f3f3f3;
+	border-radius: 0.25rem;
+	line-height: 0.6875 rem;
+}
+
+/* responsive paypal button */
+/* prevents bug causing button to get slightly cut off in mobile */
+.kv-payment-select [data-braintree-id="paypal-button"] {
+	width: 99%;
+}
+@screen md {
+	.kv-payment-select [data-braintree-id="paypal-button"] {
+		width: 250px;
+	}
+}
+
+/* Hides credit card icon in number field until credit card type is known */
+.kv-payment-select [data-braintree-id="number-field-group"]:not(.braintree-form__field-group--card-type-known) svg {
+	display: none;
+}
+
+/* Braintree Iframe is type number and inheriting some styles from kiva number input. */
+.kv-payment-select iframe[type=number] {
+	box-shadow: none;
+	padding: 0;
+	background-color: transparent;
+}
+
+/* TODO: Is this needed? */
+#payment-updating-overlay {
```

Comment:
@mcstover do you know what this is/was used for? I can't find any references to "payment-updating-overlay" in our current codebase or online.

Reply:
This can be removed. There used to be an overlay that got replaced by `kv-loading-spinner`